docs/developer-guide/create-a-distribution.md

```diff
@@ -0,0 +1,323 @@
+      },
+      "title": {
+        "type": "string",
+        "title": "Title",
+        "default": "Site",
+        "description": "A short title for the site. This will be shown as part of the title of the browser window on each page."
+      },
+      "description": {
+        "type": "string",
+        "title": "Site Description",
+        "default": "A Plone Site"
+      },
+      "default_language": {"$ref": "#/definitions/languages"},
+      "portal_timezone": {"$ref": "#/definitions/timezones"},
+      "setup_content": {
+        "type": "boolean",
+        "title": "Create Content",
+        "description": "Should example content be added during site creation?",
+        "default": false
+      }
+    }
+  },
+  "uischema": {
+  }
+}
+```
+
+````{note}
+You may have noticed the entries for both `default_language` and `portal_timezone`.
+
+```json
+      "default_language": {"$ref": "#/definitions/languages"},
+      "portal_timezone": {"$ref": "#/definitions/timezones"},
+```
+
+`plone.distribution` adds both definitions at runtime, providing a list of languages and timezones available on the installation.
+````
+
+## Add a dependency on an add-on
+
+If you want to add a Plone backend add-on to your Plone distribution, then you must perform the following steps.
+
+Add your add-on, such as `collective.person`, to your {file}`setup.py` file's `install_requires` stanza.
+
+```{code-block} python
+:emphasize-lines: 6
+
+install_requires=[
+    "setuptools",
+    "Plone",
+    "plone.distribution>=1.0.0b2",
+    "plone.api",
+    "collective.person",
+],
+```
+
+Then add it to your {file}`dependencies.zcml` file.
+
+```{code-block} xml
+:emphasize-lines: 5
+
+  <!-- List all packages your distribution depends on here -->
+  <include package="plone.volto" />
+  <include package="plone.restapi" />
+  <include package="collective.person" />
+  <include package="plone.distribution" />
+
+</configure>
+```
+
+Finally, add it to your {file}`profiles.json` file.
+
+```{code-block} json
+:emphasize-lines: 6
+
+"base": [
+  "plone.app.contenttypes:default",
+  "plone.app.caching:default",
+  "plone.restapi:default",
+  "plone.volto:default",
+  "collective.person:default",
+  "plonetheme.barceloneta:default"
+],
+```
+
+
+## Add example content
+
+The distribution loads its content from JSON data in the `content` folder.
+
+To export content from a site into this folder, use the `bin/export-distribution` script.
+
+```shell
+bin/export-distribution path/to/zope.conf Plone
+```
+
+In the example above, `Plone` is the ID of the Plone site to export.
+
+
+## Limit available distributions
+
+By default, Plone 6.1 ships with two ready-to-use distributions:
+
+[`plone.volto`](https://github.com/plone/plone.volto)
+:   Create a Plone site with the Volto frontend.
+
+[`plone.classicui`](https://github.com/plone/plone.classicui) 
+:   Create a Plone site with the Classic UI frontend.
```

Comment:
@stevepiercy I do object to switching to the package names here, because it's the distribution names, not the package names, that are used in the ALLOWED_DISTRIBUTIONS variable.

See also https://github.com/plone/documentation/pull/1733/files#r1827159965